tests/unit/src/main/java/org/mule/tck/junit4/rule/LogCleanup.java

```diff
@@ -32,20 +27,20 @@ protected void before() throws Throwable {
 
   @Override
   protected void after() {
-    clearLogsAndMDCThreadReferences();
+    clearAllLogs();
   }
 
   /**
    * Logs that are stored for later assert need to be cleared before every test
-   * 
+   * <p>
```

Comment:
This opening tag without matching closure looks suspicious. Please check.

Reply:
This tag can be used on its own, without a closing tag according to the Java Doc [documentation](https://www.oracle.com/technical-resources/articles/java/javadoc-tool.html#format), to separate paragraphs

Reply:
Sorry, I didn't know that.